### DISCUSSION TITLE
Can we show `editor` & `os` in the wakatime stats card?

### DISCUSSION THREAD

**Discussion by lemorage** [anuraghazra/github-readme-stats#4031] [Q&A] [answered]
Currently, we can show our code time using wakatime stats, but I didn't see the docs mentioning if we can also show `editor` & `os` in the wakatime stats card?
> from docs: `![Harlok's WakaTime stats](https://github-readme-stats.vercel.app/api/wakatime?username=ffflabs)`

**Comment by MrSuddenJoy** [comment]
Sure we can write code doing this. This project is MIT.

**Comment by qwerty541** [comment, marked as answer]
Hello @lemorage 

This feature is not officially supported, but you can try to deploy the following branch https://github.com/anuraghazra/github-readme-stats/pull/1202 on your own instance.
